feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/BottomSheetDaySessionsFragment.kt

```diff
@@ -97,6 +97,7 @@ class BottomSheetDaySessionsFragment : DaggerFragment() {
                     }
                 }
             groupAdapter.update(items)
+            binding.isFiltered = sessionPagesStore.filtersValue.isFiltered()
```

Comment:
Can use add changed callback for sessionPagesStore.filtersValue and do this? 👀

Reply:
@takahirom 
Thanks reviews!

Like this?

https://github.com/DroidKaigi/conference-app-2019/pull/333/commits/a36ca75f5f9e7380e2c63c09b187ac331f95ac8b